src/samplers/number.js

```diff
@@ -1,31 +1,30 @@
-const faker = require('faker');
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
 
 export function sampleNumber(schema) {
-  let res;
+  const MAX_VALUE = schema.type && schema.type === 'integer' ?
+    Number.MAX_SAFE_INTEGER : Number.MAX_VALUE;
+  const MIN_VALUE = Number.MIN_SAFE_INTEGER;
 
-  if (schema.maximum && schema.minimum) {
-    res = schema.exclusiveMinimum ? Math.floor(schema.minimum) + 1 : schema.minimum;
-    if (schema.exclusiveMaximum && res >= schema.maximum || !schema.exclusiveMaximum && res > schema.maximum) {
-      res = (schema.maximum + schema.minimum) / 2;
-    }
-    return res;
-  }
+  let schemaMin = schema.minimum + !!schema.exclusiveMinimum;
+  let schemaMax = schema.maximum - !!schema.exclusiveMaximum;
+
+  let min = schemaMin ? schemaMin : MIN_VALUE;
+  let max = schemaMax ? schemaMax : MAX_VALUE;
 
-  if (schema.minimum) {
-    if (schema.exclusiveMinimum) {
-      return Math.floor(schema.minimum) + 1;
-    } else {
-      return schema.minimum;
-    }
+  if (min > max) {
+    [min, max] = [max, min];
```

Comment:
Could you explain this line?

Reply:
Min 2
Max 3 
Exclusivemin true
Exclusivemac true

And end result is 
Min 3 max 2 
So i handle it with a swap. Its a fucking annoying edge case

Reply:
Does `Math.abs` not work? 🙃

Reply:
what do you mean?

Reply:
`Math.abs(max - min) === 1`